src/data/amd_genoa_ctrs.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
I would suggest you have a CI job that validates each and every one of the .json configs in `src/data/*.json`.  Don't have to collect any valid data, just make sure the files parse.

Probably worth providing a way for aperf to load the file and say "all good" without even trying to collect.